src/components/FilterTags/FilterTags.js

```diff
@@ -153,82 +152,108 @@ const FilterTags = () => {
-            action: "Opened filter panel"
+            action: "Toggled filter panel"
           });
         }}
-        placeholder={
-          activeFilters.length
+      >
+        <span className="p-filter-tags__active-count">
+          {activeFilters.length
             ? `Active filters: ${activeFilters.length}`
-            : "Filter models"
-        }
-      />
-
-      <div
-        className={classNames("p-card--highlighted p-filter-panel", {
+            : "Filter models"}
+        </span>
+        <i
+          className={classNames("p-icon--contextual-menu", {
+            "is-active": filterPanelVisibility
+          })}
+        ></i>
+      </button>
+      <span
+        className={classNames("p-contextual-menu__dropdown p-filter-panel", {
           "is-visible": filterPanelVisibility
         })}
+        id="filter-tags"
+        aria-hidden="true"
+        aria-label="submenu"
       >
-        {Object.entries(filters).length <= 0 && <p>Loading filters...</p>}
-        <div className="p-filter-panel__section" data-test="selected">
-          {Object.entries(filters).length > 0 && (
-            <h4 className="p-filter-panel__heading">Selected</h4>
-          )}
-          {Array.isArray(activeFilters) &&
-            Object.entries(filters).length > 0 &&
-            activeFilters.length > 0 &&
-            activeFilters.map(activeFilter => (
-              <span className="p-filter-tags__active-filter" key={activeFilter}>
-                {activeFilter.replace(":", ": ")}
-                <i
-                  className="p-icon--close"
-                  onClick={() => removeActiveFilter(activeFilter)}
+        <span className="p-contextual-menu__group">
+          {Object.entries(filters).length <= 0 && <p>Loading filters...</p>}
+          <div className="p-filter-panel__section" data-test="selected">
+            {Object.entries(filters).length > 0 && (
+              <h4 className="p-contextual-menu__dropdown__heading">Selected</h4>
+            )}
+            {Array.isArray(activeFilters) &&
+              Object.entries(filters).length > 0 &&
+              activeFilters.length > 0 &&
+              activeFilters.map(activeFilter => (
+                <span
+                  className="p-filter-tags__active-filter"
+                  key={activeFilter}
                 >
-                  Remove
-                </i>
-              </span>
-            ))}
-        </div>
-        {Object.keys(filters).map(filterBy => {
-          return (
-            filters[filterBy].length > 0 && (
-              <div key={filterBy} className="p-filter-panel__section">
-                <h4 className="p-filter-panel__heading">
-                  {pluralize(filters[filterBy].length, filterBy)}
-                </h4>
-                <ul
-                  className="p-list p-filter-panel__list"
-                  data-test={filterBy}
+                  {activeFilter.replace(":", ": ")}
+                  <i
```

Comment:
Can you make this a link element? It seems to be the only thing that doesnt make via keyboard.

Reply:
Done!